web/client/components/DashboardBuilderApp/DashboardHeader/DashboardControls/ShareDashboardModal/i18n.js

```diff
@@ -174,6 +174,7 @@ const translations: TranslationDictionary = {
       'Ocorreu um problema ao criar o link, tente novamente mais tarde.',
     'Copy URL': 'Copiar URL',
     'Dashboard Analysis Shared': 'Análise do painel compartilhada',
+    'Report Generator': 'Gerador de relat\\xF3rios',
```

Comment:
Maybe this is about to be deleted with the new release code, but is this supposed to be in Harmony?

Reply:
Great question. I'll leave this now for the sake of completeness but will likely open a separate PR with some code (and assoc. translation) redactions.